tests/graph.py

```diff
@@ -0,0 +1,37 @@
+from supporting_components.graph_io import *
+from algorithms.color_refinement import color_refinement
+
+
+def test_copy(graph):
+    """
+    Tests the copy method of the graph
+    :param graph: The graph to copy
+    :return: Whether or not the graph is copied correctly
+    """
+    copy = graph.copy()
+    return copy.is_equal(graph)
+
+
+files = ['colorref_smallexample_4_7', 'colorref_smallexample_6_15', 'colorref_smallexample_2_49']
+error_count = 0
+for file in files:
+    filename = '../test_graphs/color_refinement/' + file + '.grl'
+    with open(filename) as f:
+        L = load_graph(f, read_list=True)
+        for i in range(0, len(L[0]) - 1):
+            for j in range(i + 1, len(L[0])):
+                graph = L[0][i] + L[0][j]
+                color_refinement(graph)
+                is_equal = test_copy(graph)
+                if not is_equal:
+                    error_count += 1
+                print('---------------------------')
```

Comment:
Ik denk dat het voor deze test alleen relevant is om te weten waar het fout gaat? Dus misschien alleen die printen?

Reply:
Goed idee!